editor/src/clj/editor/properties_view.clj

```diff
@@ -160,7 +160,7 @@
                                       (properties/read-only? property))))
         cancel-fn update-prop-fn
         update-fn (fn [_]
-                    (if-let [v (field-expression/to-int (.getText text))]
+                    (when-let [v (field-expression/to-int (.getText text))]
```

Comment:
Use `when-let` to show clearly that there is no `else` branch.

Reply:
I'm still in a phase with the editor and clj when I prefer to avoid touching code I didn't write and I don't need for my fix. 
But I mentioned in comments that I was confused by that, yes `when-let` is much better in such cases